tests/mitoai_ui_tests/mitoAdvice.spec.ts

```diff
@@ -10,10 +10,10 @@ test('should display an advice message in empty code cell', async ({
   await expect
     .soft((await page.notebook.getCellLocator(0))!.getByRole('textbox'))
     .toHaveText(
-      'Start writing python or Press Ctrl + E to ask Mito AI to write code for you.'
+      'Start writing python or Press ⌘ + E to ask Mito AI to write code for you.'
```

Comment:
oo I think this is going to break on windows now. So Let's just shorten the thing to be like: toHaveText 'Start writing python or Press'